uber/models.py

```diff
@@ -1338,6 +1345,16 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_depts_ints
+
+    def assigned_and_trusted_in(self, department):
```

Comment:
it's actually used as an email filter as part of a series of (now looking at it again, slightly confusing) chipspace emails:

```
StopsEmail('MAGFest Chipspace volunteering', 'chipspace.txt',
           lambda a: (a.requested(c.JAMSPACE) or a.assigned_to(c.JAMSPACE)) and not a.assigned_to(c.CHIPSPACE))

StopsEmail('MAGFest Chipspace shifts', 'chipspace_trusted.txt',
           lambda a: a.assigned_and_trusted_in(c.CHIPSPACE))

StopsEmail('MAGFest Chipspace', 'chipspace_untrusted.txt',
           lambda a: a.has_shifts_in(c.CHIPSPACE) and not a.trusted_in(c.CHIPSPACE))
```

speaking of which I need to make sure to open that other PR.